.travis.yml

```diff
@@ -41,7 +24,11 @@ matrix:
         - cmake -H. -Bout/darwin/x64/release -DESCARGOT_HOST=darwin -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=release -DESCARGOT_OUTPUT=bin -DVENDORTEST=1 -GNinja
         - ninja -Cout/darwin/x64/release
         - cp ./out/darwin/x64/release/escargot ./escargot
-        - tools/run-tests.py --arch=x86_64 v8
+        - travis_wait tools/run-tests.py --arch=x86_64 jetstream-only-simple
```

Comment:
By experience, jetstream-only-simple requires travis_wait 30. The default 20 mins are usually not enough for that test suite.